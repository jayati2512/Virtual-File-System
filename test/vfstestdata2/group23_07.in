createvfs vfs1 1024 
movedir /dir11/dir12 /
mountvfs vfs1
makedir /dir11
makedir /dir11 dir21
makedir /dir11 dir21
makedir /dir12
makedir /dir11/dir21 dir31
listdir
makedir /dir11 dir22
movedir /dir11/dir12
listdir / 1 abc.txt
movedir /dir11/dir16 /dir12
listdir / 5 abc.txt
listdir /dir 1 output.txt
movedir /dir11/dir21 /
