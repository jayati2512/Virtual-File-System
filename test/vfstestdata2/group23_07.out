createvfs_SUCCESS
movedir_FAILURE VFS_NOT_MOUNTED 
mountvfs_SUCCESS
makedir_FAILURE: VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS 
makedir_FAILURE: VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
listdir_FAILURE VFS_INSUFFICIENT_ARGUMENTS 
makedir_SUCCESS
movedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS 
listdir_SUCCESS
movedir_FAILURE CANNOT_FIND_SPECIFIED_SOURCEDIR 
listdir_FAILURE INVALID_FLAG 
listdir_FAILURE CANNOT_FIND_SPECIFIED_PATH_OR_DIR 
movedir_SUCCESS
